dcm2niix/dcm2niix_utils.py

```diff
@@ -0,0 +1,181 @@
+"""Utility functions for pre and post dcm2niix execution."""
+
+import glob
+import logging
+import os
+import shutil
+import subprocess
+
+import nibabel as nb
+
+
+log = logging.getLogger(__name__)
+
+
+def remove_incomplete_volumes(dcm2niix_input_dir):
+    """Implement the incomplete volume correction by removal of dicom files.
+
+    Args:
+        dcm2niix_input_dir (str): The absolute path to a set of dicoms.
+
+    Returns:
+        None; removes dicoms from dcm2niix_input_dir.
+
+    """
+    n_dicom_files = len(glob.glob(dcm2niix_input_dir + "/*"))
+    log.info(
+        f"Running incomplete volume correction. {n_dicom_files} dicom files found."
+    )
+    script = f"{os.getenv('FLYWHEEL')}/fix_dcm_vols.py"
```

Comment:
Would argue that we should package the script as a function and credit the script author rather than building the image by grabbing the script from an endpoint.

Reply:
The [original dcm2niix gear](https://github.com/scitran-apps/dcm2niix) implemented this way and I've been tasked with maintaining parity. @lmperry - thoughts?

Reply:
1:1 w/Michael and decided to leave the implementation as-is for this.